core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -342,7 +343,7 @@ public PlanNode planStatement(Analysis analysis, Statement statement)
                 statement instanceof RefreshMaterializedView && analysis.isSkipMaterializedViewRefresh()) {
             Symbol symbol = symbolAllocator.newSymbol("rows", BIGINT);
             PlanNode source = new ValuesNode(idAllocator.getNextId(), ImmutableList.of(symbol), ImmutableList.of(new Row(ImmutableList.of(new Constant(BIGINT, 0L)))));
-            return new OutputNode(idAllocator.getNextId(), source, ImmutableList.of("rows"), ImmutableList.of(symbol));
+            return new OutputNode(idAllocator.getNextId(), source, ImmutableList.of(new ColumnInfo(Optional.of("catalogs"), Optional.of("schemas"), Optional.of("tables"), "rows", Optional.of("labels"))), ImmutableList.of(symbol));
```

Comment:
"catalogs", "schemas", "tables", "labels" -- these all look like fake values.
did you mean to write Optional.empty here?

Reply:
btw, it would be better not to have to change PlanNodes and LogicalPlanner, if possible.
The full column info (catalog, schema, table, name, labels) for the client protocol may come from analysis and doesn't have to be part of the plan.